internal/database/mysql.go

```diff
@@ -132,13 +152,16 @@ func (db *MySQLDatabase) AddUser(username string, email string, password string)
 	return err
 }
 
-func (db *MySQLDatabase) GetUserID(column string, row string) (int, error) {
+func (db *MySQLDatabase) GetUserID(varName string, variable string) int {
 	var id int
-	err := db.underlyingDB.QueryRow(fmt.Sprintf("SELECT id FROM users WHERE BINARY %s = ?", column), row).Scan(&id)
+	err := db.underlyingDB.QueryRow(
+		fmt.Sprintf("SELECT id FROM users WHERE BINARY %s = ?", varName), variable,
```

Comment:
This looks like a partially parameterized query, and can be insecure.

Try something like this:
query := fmt.Sprintf("SELECT id FROM users WHERE BINARY %s = ?", varName)
err := db.underlyingDB.QueryRow(query, variable).Scan(&id)


Can you confirm this needs to be fmt and not log?

Reply:
Noted, updated bugs in PR notes to deal with later

Reply:
Thank you!